src/help/components/contextual/ContextualHelpContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { Loader, SlidingPanel } from 'franklin-sites';
+import ErrorHandler from '../../../shared/components/error-pages/ErrorHandler';
+import { help as helpUrl } from '../../../shared/config/apiUrls';
+import useDataApiWithStale from '../../../shared/hooks/useDataApiWithStale';
+import { HelpEntryResponse } from '../../adapters/helpConverter';
+import { HelpEntryContent } from '../entry/Entry';
+
+import styles from './styles/contextual-help.module.scss';
+
+const ContextualHelpContainer = ({ articleId }: { articleId: string }) => {
+  const { data, loading, error, status, progress } =
+    useDataApiWithStale<HelpEntryResponse>(helpUrl.accession(articleId));
```

Comment:
If you don't use the stale flag you could just use useDataApi instead of useDataApiWithStale